src/main/java/edu/ie3/util/geo/GeoUtils.java

```diff
@@ -27,14 +27,19 @@
 import org.locationtech.jts.geom.PrecisionModel;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import tec.uom.se.ComparableQuantity;
-import tec.uom.se.quantity.Quantities;
+import tech.units.indriya.ComparableQuantity;
+import tech.units.indriya.quantity.Quantities;
 
 /** Functionality to deal with geographical and geometric information */
 public class GeoUtils {
   private static final Logger logger = LoggerFactory.getLogger(GeoUtils.class);
 
+  /* FIXME: Remove this UNIT, when the deprecated Units disappear in version 1.4 */
   /** radius of the earth in m */
+  public static final tec.uom.se.ComparableQuantity<Length> EARTH_RADIUS_OLD =
+      tec.uom.se.quantity.Quantities.getQuantity(
+          6378137.0, edu.ie3.util.quantities.dep.PowerSystemUnits.METRE);
```

Comment:
Please replace with static access ( so only METRE instead of edu.ie3.util.quantities.dep.PowerSystemUnits.METRE ), check with sonarqube for similiar listings.

Reply:
Unfortunately no alias import is available to Java. Therefore, I would opt for having the fully qualified path for those chunks, that will be removed in future version.